src/ui-game.c

```diff
@@ -216,7 +216,7 @@ struct cmd_info cmd_hidden[] =
 	{ "Steal from a monster", { 's' }, CMD_STEAL, NULL, NULL, 0, NULL, NULL, NULL, 0 },
 	{ "Walk", { ';' }, CMD_WALK, NULL, NULL, 0, NULL, NULL, NULL, 0 },
 	{ "Start running", { '.', ',' }, CMD_RUN, NULL, NULL, 0, NULL, NULL, NULL, 0 },
-	{ "Start exploring", { 'y', 'p' }, CMD_EXPLORE, NULL, NULL, 0, NULL, NULL, NULL, 0 },
+	{ "Start exploring", { 'p', 'p' }, CMD_EXPLORE, NULL, NULL, 0, NULL, NULL, NULL, 0 },
```

Comment:
Since the key is the same for both key sets, use { 'p' } rather than { 'p', 'p' }.